can/interfaces/ixxat/canlib_vcinpl.py

```diff
@@ -56,45 +56,47 @@
     log.warning("IXXAT VCI library does not work on %s platform", sys.platform)
 
 
-def __vciFormatErrorExtended(library_instance, function, HRESULT, arguments):
+def __vciFormatErrorExtended(
+    library_instance: CLibrary, function: Callable, vret: int, args: tuple
```

Comment:
`tuple` only works in python 3.9 and newer. You`could use `typing.Tuple`

Reply:
changed